### PR TITLE
Update WAFS package

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
Update WAFS package to produce high resolution WAFS output to meet 2023 ICAO milestone
1. Delay blending 0.25 trigger time by 5 minutes
2. Remove blending 1.25 job
 
Refs #2013

# Description

# Type of change

- Bug fix (fixes something broken)
- New feature (adds functionality)
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES/NO
- Does this change require a documentation update? YES/NO

# How has this been tested?

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: docs/Release_Notes.gfs.v16.3.11.md [hunk 1]
```diff
@@ -66,17 +76,22 @@ JOBS CHANGES
 PARM/CONFIG CHANGES
 -------------------
 
-* No changes from GFS v16.3.10
+* parm/wafs/wafs_gcip_gfs.cfg config file was updated
+* Change Externals.cfg to tag = "gfs_wafs.v6.3.2"
```
Comment (KateFriedman-NOAA) [inline docs/Release_Notes.gfs.v16.3.11.md]: No need to note this here, the new tag is shown above. Remove the `* Change Externals.cfg to tag = "gfs_wafs.v6.3.2"` line.

File: docs/Release_Notes.gfs.v16.3.11.md [hunk 2]
```diff
@@ -66,17 +76,22 @@ JOBS CHANGES
...
+* update ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf to ICAO2023=yes
+* update ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf to ICAO2023=yes
+* update ecf/scripts/gfs/atmos/post_processing/jgfs_atmos_wafs_gcip.ecf to ICAO2023=yes
+* In ecf/defs/gfs_v16_3.def, remove jgfs_atmos_wafs_blending task, delay trigger time of jgfs_atmos_wafs_blending_0p25 by 5 minutes
 
 FIX CHANGES
 -----------
 
-* No changes from GFS v16.3.10
+*  No changes from GFS v16.3.10
```
Comment (KateFriedman-NOAA) [inline docs/Release_Notes.gfs.v16.3.11.md]: Remove extra spaces introduced on this line.

**Review by KateFriedman-NOAA — CHANGES_REQUESTED** [review]
Updates look as discussed. Left comments to make a couple small changes to release notes. Will approve for merge into release branch when addressed. Thanks @YaliMao-NOAA !

File: docs/Release_Notes.gfs.v16.3.11.md [hunk 3]
```diff
@@ -4,7 +4,17 @@ GFS V16.3.11 RELEASE NOTES
...
+
+Additionally, EMC is updating WAFS ecf definition file to 1)trigger WAFS blending job 5 min later 
+from 4:25 to 4:30 to compensate for 5-10 min delay in receiving UKMO high resolution data;
+2)stop producing blended 1.25 deg WAFS file per ICAO milestone. 
+Both sides agreed that we sill stop waiting for data from the other side at T+4:45.
+
+Finally, NESDIS informed EMC that they will implement new global satellite composite data in Jan 2024. 
+This data set is used by one of WAFS packages. NESDIS provided a sample data set which Yali used to 
+develop an updated config file that works with current ops and to-be-implemented satellite data. 
+This updated config file will be implemented too.
```
Comment (aerorahul) [inline docs/Release_Notes.gfs.v16.3.11.md]: Following the meeting discussion on Nov 3, the NESDIS changes are not going to be included in this release.

File: docs/Release_Notes.gfs.v16.3.11.md [hunk 4]
```diff
@@ -4,7 +4,17 @@ GFS V16.3.11 RELEASE NOTES
 PRELUDE
 -------
 
-Update to the WAFS package.
+At GFS v16.3 implementation, UPP and WAFS package were updated to be ready to produce high resolution WAFS output to meet 2023 ICAO milestone with one switch. The switch was turned off at GFS v16.3. Now that UKMO has started to produce their corresponding high resolution WAFS output, EMC will work with NCO to turn on the switch.
+
+Additionally, EMC is updating WAFS ecf definition file to 1)trigger WAFS blending job 5 min later 
+from 4:25 to 4:30 to compensate for 5-10 min delay in receiving UKMO high resolution data;
+2)stop producing blended 1.25 deg WAFS file per ICAO milestone. 
+Both sides agreed that we sill stop waiting for data from the other side at T+4:45.
```
Comment (aerorahul) [inline docs/Release_Notes.gfs.v16.3.11.md]:
```suggestion
Additionally, EMC is updating WAFS ecf definition file to:

1. trigger WAFS blending job 5 min later from 4:25 to 4:30 to compensate for 5-10 min delay in receiving UKMO high resolution data
2. stop producing blended 1.25 deg WAFS file per ICAO milestone. 

Both sides agreed that we sill stop waiting for data from the other side at T+4:45.
```

**Review by aerorahul — COMMENTED** [review]
Some comments for your consideration.

**Comment by YaliMao-NOAA** [comment]
Steven didn't approve combining WAFS blending update and GCIP parm update this morning. I am going to make the change to switch back to gfs_wafs.v6.3.1

**Comment by YaliMao-NOAA** [comment]
The current UK data at DCOM is /lfs/h1/ops/**prod**/dcom/20231103/wgrbbul/ukmet_wafs
The parallel UK data will be /lfs/h1/ops/**para**/dcom/20231103/wgrbbul/ukmet_wafs, which is going to be created Nov 6.

My question is how the parallel run will point to the para DCOM, not the prod DCOM. In global ecflow, which variable is controlling this?

**Review by KateFriedman-NOAA — APPROVED** [review]

File: ecf/defs/gfs_v16_3.def [hunk 5]
```diff
@@ -1584,12 +1584,9 @@ suite gfs_v16_3
                     trigger ../../post/jgfs_atmos_post_f000 == complete
                   task jgfs_atmos_wafs_grib2_0p25
                     trigger ../../post/jgfs_atmos_post_f036 == complete
-                  task jgfs_atmos_wafs_blending
```
Comment (YaliMao-NOAA) [inline ecf/defs/gfs_v16_3.def:1587]: @KateFriedman-NOAA 1.25 deg WAFS blending needs to be added back for https://github.com/NOAA-EMC/global-workflow/pull/2100

Comment (KateFriedman-NOAA) [reply ecf/defs/gfs_v16_3.def:1587]: @YaliMao-NOAA It's in the `gfs_v16_3.def` file for the `release/gfs.v16.3.11` branch: https://github.com/NOAA-EMC/global-workflow/blob/release/gfs.v16.3.11/ecf/defs/gfs_v16_3.def#L1587

It's also in the def file in the `release/gfs.v16.3.12` branch. I undid the WAFS changes in the `release/gfs.v16.3.11` branch and those changes were inherited by the `release/gfs.v16.3.12` branch.

I also see it in the `gfs.v16.3.11` para package on WCOSS2:
```
kate.friedman@clogin06> grep jgfs_atmos_wafs_blending /lfs/h1/ops/para/packages/gfs.v16.3.11/ecf/defs/gfs_v16_3.def
                  task jgfs_atmos_wafs_blending
                  task jgfs_atmos_wafs_blending_0p25
                  task jgfs_atmos_wafs_blending
                  task jgfs_atmos_wafs_blending_0p25
                  task jgfs_atmos_wafs_blending
                  task jgfs_atmos_wafs_blending_0p25
                  task jgfs_atmos_wafs_blending
                  task jgfs_atmos_wafs_blending_0p25
```
...and I see the job in the `gfs.v16.3.11` prod package on Dogwood that the NCO SPA just put into production.

I think we're good, please let me know if I'm missing something.

Comment (KateFriedman-NOAA) [reply ecf/defs/gfs_v16_3.def:1587]: Ignore my prior reply, I see it's missing now but thankfully the ops version has it in `/lfs/h1/ops/prod/packages/gfs.v16.3.11/ecf/defs/gfs_v16_3.def`:
```
                  task jgfs_atmos_wafs_blending
                    trigger ./jgfs_atmos_wafs_grib2 == complete
                    time 04:33
```

Comment (YaliMao-NOAA) [reply ecf/defs/gfs_v16_3.def:1587]: @KateFriedman-NOAA That's good. Cactus hasn't synchronized gfs.v16.3.11 yet so I can't compare the file.

Comment (KateFriedman-NOAA) [reply ecf/defs/gfs_v16_3.def:1587]: @YaliMao-NOAA You can compare the para install of that version on Cactus: `/lfs/h1/ops/para/packages/gfs.v16.3.11/ecf/defs/gfs_v16_3.def`
That's the package that was moved to prod on Dogwood.

Comment (YaliMao-NOAA) [reply ecf/defs/gfs_v16_3.def:1587]: @KateFriedman-NOAA Good. The para version on Cactus includes 1.25 deg WAFS blending. Should I withdraw PR #2113?

Comment (KateFriedman-NOAA) [reply ecf/defs/gfs_v16_3.def:1587]: > Should I withdraw PR https://github.com/NOAA-EMC/global-workflow/pull/2113?

@YaliMao-NOAA Nope, all good! We were missing that change on our end so I've already merged that PR and we're good now. Thanks for catching that in our branch!

**Pull request merged**